test/test_utils/__init__.py

```diff
@@ -80,6 +80,12 @@ def is_tf20(image_uri):
     return bool(re.search(r'2\.0\.\d+', image_uri))
 
 
+def above_tf23(image_uri):
```

Comment:
we should use packaging or something similar to check package versions. I.e. if this was tf2.15, it would not satisfy this condition

Reply:
I can modify it to skip if it is under 2.3, so future verisons could pass